storage/backup/backup-cli/src/metadata/mod.rs

```diff
@@ -93,13 +145,29 @@ pub struct EpochEndingBackupMeta {
     pub manifest: FileHandle,
 }
 
+#[derive(Clone, Deserialize, Serialize, Eq, PartialEq, Ord, PartialOrd)]
+pub struct StateSnapshotBackupMetaRange {
+    pub first_epoch: u64,
+    pub last_epoch: u64,
+    pub first_version: Version,
+    pub last_version: Version,
+    pub backup_metas: Vec<StateSnapshotBackupMeta>,
+}
+
 #[derive(Clone, Deserialize, Serialize, Eq, PartialEq, Ord, PartialOrd)]
 pub struct StateSnapshotBackupMeta {
     pub epoch: u64,
     pub version: Version,
     pub manifest: FileHandle,
 }
 
+#[derive(Clone, Deserialize, Serialize, Eq, PartialEq, Ord, PartialOrd)]
+pub struct TransactionBackupMetaRange {
+    pub first_version: Version,
```

Comment:
Do we backup transactions across epoch? Would it be possible to have epoch information here?

Reply:
This struct is no longer used. ..